run_elixir

```diff
@@ -2,10 +2,9 @@
 
 export TIMEFORMAT=%R
 cd elixir/
-mix compile.protocols
```

Comment:
I kept this line because I didn't want the compilation to be included in the `time` call.